Automatic/keepassxc-protector.ini

```diff
@@ -0,0 +1,20 @@
+[Command]
+Automatic=true
+Command="
+    copyq:
+    
+    if ( ! isClipboard() ) {
+    \tfail();
+    }
+    
+    var title = currentWindowTitle();
```

Comment:
This won't work if you have "Minimize when copying to clipboard" KeePassXC option enabled.

It's better to match window title by setting `Window` field (as in `Ignore *"Password"* window` predefined command).